Reorganize Eth1-specs Repo & Add Network Upgrades Info (#24)
ethereum/eth1.0-specs: network-upgrades/README.md

# Network Upgrades Specifications

## Description

> This folder provides the specifications for the various network upgrades to Ethereum. It contains specifications for the client integration testnets which are set up during the testing phase of upgrades, for the network upgrades which will be deployed to the Ethereum mainnet, as well as retrospectives/postmortems on network upgrade-related incidents.

Reviewer: ```suggestion
This folder provides the specifications for the various network upgrades to Ethereum 1.0. It contains specifications for the client integration testnets which are set up during the testing phase of upgrades, for the network upgrades which will be deployed to the Ethereum mainnet, as well as retrospectives/postmortems on network upgrade-related incidents.
```

Author: Good catch!

> In an effort to separate this process from the EIP process itself, three stages have been devised to move EIPs from specifications to being deployed on mainnet: Considered for Inclusion (CFI), Client Integration Testnets, and Mainnet. Below, we explain the process for EIPs to move through these stages.

Reviewer: ```suggestion
In an effort to separate this process from the EIP standardization process itself, three stages have been devised to move EIPs from specifications to being deployed on the mainnet: Considered for Inclusion (CFI), Client Integration Testnets, and Mainnet. Below, we explain the process for EIPs to move through these stages.
```

Author: Noted.

## Definitions

> **Considered for Inclusion**: Signals that client developers are generally positive towards the idea, and that, assuming it meets all the requirements for mainnet inclusion, it could potentially be included in a network upgrade. This means the EIP may be included in client integration testnets. It is similar to "concept ACK" in other open source projects, and is not sufficient to result in deployement to mainnet.

Reviewer: ```suggestion
**Considered For Inclusion**: CFI approved signals that client developers are generally positive towards the idea, and that, assuming it meets all the requirements for the mainnet inclusion, it could potentially be included in a network upgrade. This means the EIP may be included in the client integration testnets. It is similar to "concept ACK" in other open-source projects, and is not sufficient to result in deployment to the mainnet.
```

Author: I'm not sure we should call it "CFI approved": we can just say that when something is considered for inclusin, then it signals that client devs like it. 

Also don't think all the "the" are needed.

Reviewer: Just suggestions, fine with the original texts.

**Client Integration Testnets**: Short-lived integration testnets which are stood up to test cross-client implementations of certain EIPs. This does not guarantee mainnet deployement, and for some EIPs with small or non-applicable changes, this step may be skipped.

> **Mainnet**: Signals that client developers with to include the EIP into an upgrade to the public Ethereum networks (i.e. testnets such as Goerli, Ropsten and Rinkeby and mainnet).

Reviewer: ```suggestion
**Mainnet**: Signals that client developers wish to include the EIP into an upgrade to the public Ethereum networks (i.e. testnets such as Goerli, Ropsten and Rinkeby and mainnet).
```

Author: Noted!

## Process

### Preconditions for Consideration

_See [EIP-1](https://eips.ethereum.org/EIPS/eip-1#core-eips) for more context._

* You have vetted your idea and created an EIP by opening a PR against the [ethereum/eips](https://github.com/ethereum/EIPs/pulls) repository;
* The EIP is a "Core" EIP;
* The EIP's status is "Draft" or "Review";
* [Optional, but strongly encouraged] You have socialized your EIP and collected some initial feedback in the `discussions-to` links from your EIP.

> ### Getting the "Considered for Inclusion" (CFI) Status

Reviewer: ```suggestion
### Getting the "Considered For Inclusion" (CFI) Status
```

Author: Going to go with Considered for Inclusion as you suggested above :-)
